incubator/hnc/cloudbuild.yaml

```diff
@@ -12,7 +12,16 @@ steps:
     cd multi-tenancy
     echo Checking out hnc-$_HNC_IMG_TAG
     git checkout hnc-$_HNC_IMG_TAG
-# Build the manifests and the kubectl plugin
+# go fmt enforcement
+- name: golang:1.14
```

Comment:
We only use this file (and Cloud Build) when we're generating an official release. The file that gets run on every commit by Github is https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/hnc/hack/ci-test.sh - can you please move this logic there?

Reply:
My fault! Doing